frontend/packages/network-attachment-definition-plugin/src/plugin.ts

```diff
@@ -0,0 +1,51 @@
+import * as _ from 'lodash';
+import {
+  Plugin,
+  ResourceNSNavItem,
+  ResourceListPage,
+  ModelFeatureFlag,
+  YAMLTemplate,
+  ModelDefinition,
+  RoutePage,
+} from '@console/plugin-sdk';
+import * as models from './models';
+
+type ConsumedExtensions =
+  | ResourceNSNavItem
+  | ResourceListPage
+  | ModelFeatureFlag
+  | YAMLTemplate
+  | ModelDefinition
+  | RoutePage;
+
+const plugin: Plugin<ConsumedExtensions> = [
```

Comment:
shouldn't we also define a flag for NetworkAttachmentDefinitionModel and show the  extension only in that case?

Reply:
Done!